creator-node/src/routes/healthCheck.js

```diff
@@ -1,13 +1,102 @@
+const disk = require('diskusage')
+const axios = require('axios')
+const ethSigUtil = require('eth-sig-util')
+
 const { handleResponse, successResponse, errorResponseServerError } = require('../apiHelpers')
+const { authMiddleware, crossCnodeAuth } = require('../middlewares')
 const { sequelize } = require('../models')
 const config = require('../config.js')
 const versionInfo = require('../../.version.json')
-const disk = require('diskusage')
+const utils = require('../utils')
 
 const MAX_DB_CONNECTIONS = 90
 const MAX_DISK_USAGE_PERCENT = 90 // 90%
 
 module.exports = function (app) {
+  /** Call this to tell CN to call another CN's /permission route to test cross-CN auth */
+  app.get('/test', handleResponse(async (req, res) => {
+    const redisClient = req.app.get('redisClient')
+
+    const cnodeURL = req.query.cnodeURL // string
```

Comment:
we should standardize what we use to look up cnodes by and i think it should be spId cause it doesn't change and we can look it up on the. eth chain easily

Reply:
per discussion, going to standardize all to spID